bin/webpack-dev-server.js

```diff
@@ -294,6 +296,25 @@ function processOptions(wpOpt) {
 	if(argv["open"])
 		options.open = true;
 
+	// Kind of weird, but ensures prior behavior isn't broken in cases
+	// that wouldn't throw errors. E.g. both argv port and options port
+	// were specified, but since argv port is 8080, options port will be
+	// used instead.
+	options.port = argv.port === DEFAULT_PORT ? options.port || argv.port : argv.port;
```

Comment:
There seems to be an error here. If I set the port via `options`, after this line it results in `options.port` being `undefined`.